rust/pkg/cardano_serialization_lib.js.flow

```diff
@@ -3219,6 +3257,18 @@ declare export class Pointer {
    */
   static new(slot: number, tx_index: number, cert_index: number): Pointer;
```

Comment:
Mark `.new` as deprecated, mention that it "uses simplified slot number type", and mention to "use `.new_pointer` instead